build_tools/packaging/python/templates/rocm/src/rocm_sdk/__main__.py

```diff
@@ -127,6 +140,12 @@ def main(argv: list[str] | None = None):
     )
     targets_p.set_defaults(func=_do_targets)
 
+    # 'init' subcommand.
+    init_p = sub_p.add_parser(
+        "init", help="Expand devel contents to initialize rocm[dev]"
+    )
```

Comment:
The extra is named "devel", not "dev".
```suggestion
    init_p = sub_p.add_parser(
        "init", help="Expand devel contents to initialize rocm[devel]"
    )
```